### DISCUSSION TITLE
Welcome to public_h-60 Discussions!

### DISCUSSION THREAD

**Discussion by A26mike** [Project-Hatchet/public_h-60#16] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by sevn10** [comment]
I'm super happy to see this finally being worked on again. I was in the discord until discord deleted my account so I kind of lost track where things were for the project. Anyhow, when working on a simple script to add the door numbers to ACE actions I noticed I never sent a number 4 in the zip I gave Yax. I've made an updated zip including the missing .paa if you want to include it. I would also be willing to contribute my skins to the main mod too. 
[VTX_Door_Numbers_Update.zip](https://github.com/Project-Hatchet/public_h-60/files/6039990/VTX_Door_Numbers_Update.zip)
